### DISCUSSION TITLE
Use markdown extensions in i11ty

### DISCUSSION THREAD

**Discussion by hirunatan** [11ty/eleventy#1636] [Q&A] [answered]
Hi. I'd like to ask if I can use some extensions of the markdown template language inside i11ty. For example to include ascii-art diagrams with plantuml, ditaa or kroki and convert them to images in the final html result.

https://github.com/mikitex70/plantuml-markdown

Is it possible? There are other extensions that I'd like to use, like those for including samples of code into the document, or generate complex tables.

**Comment by nhoizey** [comment, marked as answer]
You can use markdown-it plugins:
https://www.11ty.dev/docs/languages/markdown/#add-your-own-plugins

Looks like there are a few for PlantUML:
https://www.npmjs.com/search?q=keywords%3Amarkdown-it-plugin%20keywords%3Aplantuml

**Reply by hirunatan** [reply]
Yes! That's probably what I was looking for. Thank you very much!
